tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+import pytest as pytest
+
+from rocket.rocket import Rocket
+
+
+@pytest.fixture()
+def python_rocket():
```

Comment:
Lets type annotate everywhere possible, long term it really pays off for quality assurance. But in the short term it already pays off for readability :).

Reply:
👍